hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
I just wanted to point out that there is already an unpredictable behaviour considering included files.